### PR TITLE
Feature/catalog

### PR DESCRIPTION
**Pull request by zrichardet**
Unable to get The Product Cards to Load on the StoreFront

### CODE REVIEW

File: web/gatsby-config.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ require('dotenv').config({
 
 module.exports = {
   siteMetadata: {
-    title: 'Gatsby starter ecommerce',
-    author: 'Parminder Sanghera',
-    description: 'A starter e-commerce site made using Gatsby.',
+    title: 'Bella Floral',
+    author: 'Zach Richardet',
+    description: '',
     siteUrl: 'https://parmsang.github.io/gatsby-starter-ecommerce/',
```
Comment (tetriscode) [inline web/gatsby-config.js]: `store.bellafloral.co`

File: web/gatsby-config.js [hunk 2]
```diff
@@ -26,13 +26,21 @@ module.exports = {
         name: 'pages',
       },
     },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `florists`,
```
Comment (tetriscode) [inline web/gatsby-config.js]: `florist`

File: web/src/pages/index.js [hunk 3]
```diff
@@ -45,8 +38,8 @@ const StoreIndex = ({location}) => {
   `)
 
   const siteTitle = get(data, 'site.siteMetadata.title')
-  const products = get(data, 'allMoltinProduct.edges')
-  const filterProductsWithoutImages = products.filter(v => v.node.mainImageHref)
+  const products = get(data, 'allFloristsJson.edges')
+  const filterProductsWithoutImages = products.filter(v => v.node.skus.image)
```
Comment (tetriscode) [inline web/src/pages/index.js]: `ProductList` needs to have the ability to support the result of your new graphql query. It is expecting `meta` and `mainImage`.

**Comment by zrichardet** [comment]
Screen Shot 2020-02-28 at 8 30 39 AM
Floral Storefront now loads products and images.

**Pull request merged**